bin/dispatcher.py

```diff
@@ -85,13 +85,21 @@ def main():
                         help="Verbosity (between 1 and 2 occurrences with more leading to more "
                         "verbose logging). WARN=0, INFO=1, "
                         "DEBUG=2. This is overridden by the log config file if specified.")
+    parser.add_argument(
+        "-p", "--publish-port", type=int, dest="pub_port",
+        help="Publish messages for dispatched files on this port. "
+        "Default: no publishing.")
```

Comment:
Can you set this to 0 if -p is provided but not number is given ? (random port)

Reply:
The option requires an argument, so plain `-p` won't work.

Reply:
I think there is a parameter for this in argparse.

Reply:
Updated. Calling `dispatcher.py config.yaml -p` will result in
`Namespace(config_file='config.yaml', log=None, log_config=None, pub_nameservers=None, pub_port=0, verbosity=0)`.

Calling `dispatcher.py config.yaml` will result in
`Namespace(config_file='config.yaml', log=None, log_config=None, pub_nameservers=None, pub_port=None, verbosity=0)`

Is this what you suggested?

Reply:
yes, perfect, thanks !